src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +20,11 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+        return movieDao.get(id).orElseThrow(() -> new DataProcessingException("No such movie"));
```

Comment:
According to the checklist, the `get()` method on dao layer should return `Optional`, and the entity should be returned on service layer. Here, the `get()` method on the dao layer is used correctly, but the exception message should be more specific to the entity and method. Consider specifying the entity name and the identifier for which the entity was not found in the exception message.